### PR TITLE
Release Notes for DTM 10.16

### PR DESCRIPTION
**Pull request by spau6**
@lra-sag, @rssag, @MWindel, Please review the Release notes for 10.16 version of DTM

### CODE REVIEW

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:23]:
```suggestion
<td>The DTM application is integrated with Gainsight to capture the data.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:29]: what does "graceful" mean?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:35]:
```suggestion
<td>The on delete operation is shown in the Property library page confirmation pop-up.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:41]:
```suggestion
<td>Implemented a maximum file size field for custom properties of type file.</td>
```
please check if this is the meaning that was intended.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:47]: see above, what does "graceful" mean?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:53]: see above, what does "graceful" mean?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:59]:
```suggestion
<td>An alert save pop-up is displayed in the localization page.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:81]:
```suggestion
<td>The Import Assets button is not visible for the last asset in the hierarchy now.</td>
```
this sounds a bit like it describes the situation prior to fixing - should it really be invisible?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:87]:
```suggestion
<td>The loading message in the localization page is corrected.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:93]:
```suggestion
<td>An issue observed during hierarchy creation is fixed.</td>
```
What was the issue?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:99]:
```suggestion
<td>An issue while creating subassets when there are both required and optional child asset types at same level is fixed.</td>
```
What was the issue?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:105]:
```suggestion
<td>Changes made in the asset type page are getting saved now.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:111]:
```suggestion
<td>Provided a fix for the Add child asset window which became unresponsive for assets that had a custom property deleted from the backend.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:117]:
```suggestion
<td>A warning message is now displayed on deletion of a custom property from the backend.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:123]:
```suggestion
<td>The properties tab is not displayed when there are no properties defined.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:135]:
```suggestion
<td>Users are now able to delete the uploaded file in custom properties of an asset.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:28]: this line is repeated for many entries. could we just place it once at the top instead?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:86]: see above, added for every entry instead of "once"

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:147]:
```suggestion
<td>Fixed localization issues related to the bulk import feature.</td>
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (MWindel) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:28]: I agree.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (MWindel) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:41]:
```suggestion
<td>Implemeted maximum file size field for the custom property "file type".</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (MWindel) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:59]: @spau6 What is meant with "alert save"?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (MWindel) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:81]:
```suggestion
<td>Th **Import assets** button is no longer visible for last asset in the hierarchy.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:29]: Basically we are displaying a warning message stating that property not found

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:47]: Editing an already used AssetType will show a popup on click of save button stating below
"This asset type is used by one or more existing assets and they may not reflect this change. Do you want to proceed?"
User can click on either "Cancel" or "Confirm" button

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:53]: Editing an already used custom property will show a popup on click of save button stating below
"This custom property is associated with one or more asset types. Any existing assets of those types may not reflect this change. Do you want to proceed?"
User can click on either "Cancel" or "Confirm" button

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (lra-sag) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:47]:
```suggestion
<td>Added a confirmation popup which notifies user that the changes made to asset type will not be reflected in the assets that are currently associated with it</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (lra-sag) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:53]:
```suggestion
<td>Added a confirmation popup which notifies user that the changes made to the custom property will not be reflected in the assets that are currently associated with it's asset type</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Placed as part of heading

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Placed as part of heading

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:25]:
```suggestion
<td>Added a warning notification if a custom property associated with an asset is deleted using Cumulocity REST APIs when an asset is viewed.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:30]:
```suggestion
<td>A confirmation pop-up is shown in the <b>Property library</b> page for every delete operation.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:40]:
```suggestion
<td>Added a confirmation popup which states that the changes made to the asset type will not be reflected in the assets that are currently associated with it.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:45]:
```suggestion
<td>Added a confirmation popup which states that the changes made to the custom property will not be reflected in the assets that are currently associated with it's asset type.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (rssag) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:69]:
```suggestion
<td>The <b>Import Assets</b> button is not visible for the last asset in the hierarchy now.</td>
```

**Review by rssag — CHANGES_REQUESTED** [review]

**Review by rssag — APPROVED** [review]

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:30]:
```suggestion
<td>A confirmation popup is shown in the <b>Property library</b> page for every delete operation.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:35]:
```suggestion
<td>Implemented a maximum file size field for custom properties of type "file".</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:50]:
```suggestion
<td>An alert save popup is displayed in the localization page.</td>
```

Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:50]: localization page > is this the name of a page? Then it should be <b>Localization</b> page

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:7]:
```suggestion
### Improvements
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:20]: I don´t think that users know what Gainsight is. Please explain a bit more what this means and what is captured.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:25]:
```suggestion
<td>Added a warning notification which shows up if a custom property associated with an asset is deleted using Cumulocity IoT REST APIs while an asset is viewed.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:50]: What is an "alert save popup"?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:56]:
```suggestion
### Fixes
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:69]:
```suggestion
<td>The <b>Import assets</b> button is no longer visible for the last asset in the hierarchy.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:74]:
```suggestion
<td>The loading message in the localization page has been corrected.</td>
```

Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:74]: Should this be <b>Localization</b> page?

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:89]:
```suggestion
<td>Changes made in the <b>Asset type</b> page are now saved.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:104]:
```suggestion
<td>The <b>Properties</b> tab is not displayed when there are no properties defined.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:94]:
```suggestion
<td>A fix has been fix for the <b>Add child asset</b> window which became unresponsive for assets that had a custom property deleted from the backend.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:114]:
```suggestion
<td>Users can now delete the uploaded file in the custom properties of an asset.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:119]:
```suggestion
<td>Fixed an issue where a bulk import failed if the complex property contains a duplicate label.</td>
```

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:129]:
```suggestion
<td>The creation of assets via bulk import is now triggered in the Gainsight application.</td>
```

Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:129]: Again, I don´t think that Gainsight is a term familiar to the user.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:94]: I think you mean"A fix has been provided" not "A fix has been fix"

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Pop up which informs user about their unsaved changes

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Done

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Yes, modified.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Explained about it

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (spau6) [reply content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md]: Yes, modified.

File: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md
Comment (BeateRixen) [inline content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md:50]:
```suggestion
<td>An alert popup which warns the user about unsaved changes is displayed in the <b>Localization</b> page.</td>
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@spau6 Please have one approval by a member of your team before merging.

**Review by lra-sag — APPROVED** [review]

**Pull request merged**
